src/axom/inlet/docs/sphinx/index.rst

```diff
@@ -38,6 +38,19 @@ Requirements
 * (Optional) Lua - Inlet provides a Lua reader class that assists in Lua input files
 
 
+Glossary
+--------
+
+* ``inlet::Table``: Internal nodes of the Inlet hierarchy
+* ``inlet::Field``: Terminal nodes of the Inlet hierarchy that store primitive types (bool, int, string, double)
+* ``inlet::Function``: Terminal nodes of the Inlet hierarchy that store function callbacks
+* ``inlet::Proxy``: Provides type-erased access to data in the Inlet hierarchy - can
+  refer to either a ``Table``, ``Field``, or ``Function internally
```

Comment:
"``Function" -> "``Function``"

Reply:
Well the formatting messed that up but you can figure out what I'm saying.

Reply:
Good catch, I couldn't tell what was messing up my syntax highlighting